python/gink/impl/lmdb_store.py

```diff
@@ -973,8 +1005,7 @@ def _add_entry(
         except BadValsizeError:
             raise BadValsizeError("Max key size for LMDB is 511 bytes.")
         entries_loc_key = bytes(LocationKey(entry_muid, entry_muid))
-        if builder.behavior in (EDGE_TYPE, SEQUENCE):
-            txn.put(entries_loc_key, serialized_placement_key, db=self._locations)
+        txn.put(entries_loc_key, serialized_placement_key, db=self._locations)
```

Comment:
_remove_entry uses locations to remove from placements. Let me know if this is a bad way of doing this.

Reply:
Seems sensible.